victoria_bank.rb

```diff
@@ -0,0 +1,151 @@
+require 'watir'
+require 'nokogiri'
+require 'pry'
+require 'json'
+require 'rubocop'
+require 'open-uri'
+require_relative 'transactions.rb'
+require_relative 'accounts.rb'
+
+class VictoriaBank  
+  def get_user_credentials
+    puts "Enter your username"
+    @username = gets.chomp
+    puts "Please enter your password"
+    @password = gets.chomp
+  end
+
+  def set_user_credentials
+    @browser.text_field(class: 'username').set(@username)
+    @browser.text_field(id: 'password').set(@password)
+    @browser.button(class: 'wb-button').click
+    validate_login
+  end
+
+  def validate_login
+    Watir::Wait.until do
+      if @browser.div(class: "error", text: "Invalid username or password").present?
+        puts "Inlavid username or password"
+        break
+      end
+      @browser.div(class: "contracts").present?
+    end
+  end
+
+  def init_session
+    @browser = Watir::Browser.new
+    @browser.goto "https://web.vb24.md/wb/#login"
+    Watir::Wait.until{ @browser.input(class: 'username').present? }
+  end
+
+  def run
+    init_session
+    get_user_credentials
+    set_user_credentials
+    access_accounts
+    access_transactions
+    sort
+    export_to_json
+  end
+
+  def access_accounts
+    divs = @browser.divs(class: 'main-info')
+    @account_details = []
+    divs.each do |element|
+      accounts_html = Nokogiri::HTML(element.html)
+      parse_acc(accounts_html)
+    end
+    @account_details
+  end
+
+  def access_transactions
+    @browser.a(class: 'menu-link').click
+    sleep 0.5
+
+    @account_details.each do |acc|
+      @browser.a(class: 'chosen-single').click
+      Watir::Wait.until{ @browser.span(class: 'contract-name', text: acc.name).present? }
+      @browser.span(class: 'contract-name', text: acc.name).click
+      sleep 1
+      set_date
+      next if @browser.h1(text: "No operations found").present?
+
+      acc.transactions = get_trans
+    end
+  end
+
+  def parse_acc(accounts_html)
+    name     = accounts_html.at_css('a.name').text
+    balance  = accounts_html.at_css('span.amount').text
+    currency = accounts_html.at_css('span.currency').text
+    accounts_html.css('div')[3].content.empty? ? nature = 'Account' : nature ='Card Account'
+    account  = Accounts.new(name, currency, balance, nature)
+    @account_details << account
+  end
+
+  def parse_transaction(transactions_html)
+    description         = transactions_html.css('h1').text
```

Comment:
use `.strip` method for remove unnecessary space in description  
 `transactions_html.css('h1').text.strip`